draft-ietf-core-problem-details.md

```diff
@@ -386,16 +392,15 @@ entries are defined specifically to the identifier of that Custom Problem Detail
 entry, the documentation of which defines these internal entries,
 typically chosen to address a given application domain.
 
-Consumers of a Concise Problem Details data item MUST ignore any Custom
-Problem Detail entries, or keys inside the Custom Problem Detail
-entries, that they do not recognize; this allows Custom Problem Detail
-entries to evolve and include additional information in the future.
+When evolving Custom Problem Detail entry definitions, the
+"ignore-unknown rule" discussed in the introduction to
+{{sec-new-attributes}} can help with including additional information.
```

Comment:
It sounds like an pretty intricate way of just saying:
```suggestion
The "ignore-unknown rule" discussed in the introduction to
{{sec-new-attributes}} applies fully.
```
no? 😄

Reply:
Well, this sentence is a lead-in to specifically discussing ignore-unknown *inside* a CPDe.
So I don't think the simplified form helps as much.

Reply:
Removing _"fully"_ from the suggested text could do the trick?

Stepping back a bit: I have troubles making sense of _"[...] can help with including additional information."_.

I think what we are trying to tell the reader is: "apply ignore-unknown when dealing with extensions found in CPDe".  If so, the way the text flows is not conducive to quick and unambiguous grasp of the concept.

Reply:
I think we are trying to explain you don't have to overdo the evolution of a CPDe because you always can make a new one.

Reply:
Then we probably should say exactly so :-)

Reply:
I think we do.  Note that this is text that is directed at the person who wants to evolve a CPDe definition; if it were about handling evolved definitions, it would indeed be redundant.

I tried to make this context more accessible in adb7076